thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/client/comparison/TimeOnTimeResponseParser.java

```diff
@@ -316,7 +316,19 @@ private void parseGroupByTimeDimensionResponse() {
           otherBuilder.addMetric(metricFunctions.get(i).getMetricName(), otherBaseline[i],
               otherCurrent[i]);
         }
-        rows.add(otherBuilder.build());
+
+        if (otherBuilder.row.getMetrics().size() > 0) {
+          // Add only a non zero metric
+          boolean addMetric = false;
+          for (Row.Metric metric : otherBuilder.row.getMetrics()) {
+            if (metric.getCurrentValue() > 0.0 || metric.getBaselineValue() > 0.0) {
```

Comment:
these are just values, not the change (current vs baseline). So yes, these should be non negative.